jobs/openshift4/cluster/create/Jenkinsfile

```diff
@@ -0,0 +1,180 @@
+                dir(".") {
+                    rootWorkspaceDir = sh (
+                        script: "pwd",
+                        returnStdout: true
+                    ).trim()
+                    installConfigDirectory = rootWorkspaceDir + "/rhmi-jenkins/" + clusterInfoDirectory + "/"
+                    sh "./openshift-install create cluster --dir=${installConfigDirectory} --log-level=${LOG_LEVEL} 2>&1 | tee log.file"
+                    String clusterCreationOutput = readFile("log.file")
+                    sh "mv log.file ${installConfigDirectory}"
+                }
+            } // stage
+            
+            stage("Push config files to GitLab") {
+                withCredentials([string(credentialsId: "qe-gitlab-access-token", variable: "GITLAB_ACCESS_TOKEN")]) {
+                    dir("rhmi-jenkins") {
+                        sh(
+                            returnStdout: false,
+                            script: """
+                                git remote add jenkins https://oauth2:${GITLAB_ACCESS_TOKEN}@gitlab.cee.redhat.com/integreatly-qe/rhmi-jenkins.git
+                                git add -A
+                                git commit -m 'cluster created: ${clusterName}'
+                                git push jenkins master
+                            """
+                        )
+                    }
+                }
+            } // stage
+            
+            stage("Install dedicated-admin operator") {
+                String installationLog = sh (
+                    script: "cat ${installConfigDirectory}log.file",
+                    returnStdout: true
+                ).trim()
+                if (installationLog.contains("Install complete")) {
+                    adminPassword = sh (
+                        script: "cat ${installConfigDirectory}auth/kubeadmin-password",
+                        returnStdout: true
+                    ).trim()
+                    dir(".") {
+                        sh(
+                            returnStdout: false,
+                            script: """
+                                sudo yum install -y httpd-tools # to be able to use htpasswd util
+                                wget -q https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux-4.1.11.tar.gz
+                                tar xvzf openshift-client-linux-4.1.11.tar.gz
+                                
+                                ./oc version
+                                ./oc login https://api.${clusterName}.${baseDomain}:6443 --insecure-skip-tls-verify=true -u kubeadmin -p ${adminPassword}
+                            """
+                        )
+                    }
+                    dir("integreatly-operator") {
+                        git branch: "master", url: "https://github.com/integr8ly/integreatly-operator.git"
+                    }
+                    dir("."){
```

Comment:
Why do you need to wrap these in a dir("."), do the internal commands require it?